v20.1/backup-and-restore.md

```diff
@@ -1,67 +1,96 @@
 
 ## Perform Enterprise backup and restore
 
 If you have an [Enterprise license](enterprise-licensing.html), you can use the [`BACKUP`][backup] statement to efficiently back up your cluster's schemas and data to popular cloud services such as AWS S3, Google Cloud Storage, or NFS, and the [`RESTORE`][restore] statement to efficiently restore schema and data as necessary.
 
-### Manual full backups
+{{site.data.alerts.callout_success}}
+We recommend [automating daily backups of your cluster](#automated-full-and-incremental-backups). To automate backups, you must have a client send the `BACKUP` statement to the cluster. Once the backup is complete, your client will receive a `BACKUP` response.
+{{site.data.alerts.end}}
+
+### Full backups
+
+In most cases, **it's recommended to take full nightly backups of your cluster**. A full cluster backup allows you to do the following:
+
+- Restore a full cluster
+- Restore database(s) from the cluster
+- Restore table(s) from the cluster
+
+To do a full cluster backup, use the [`BACKUP`](backup.html) statement:
+
+{% include copy-clipboard.html %}
+~~~ sql
+> BACKUP TO '<backup_location>';
+~~~
+
+If it's ever necessary, you can use the [`RESTORE`][restore] statement to restore your full cluster:
+
+{% include copy-clipboard.html %}
+~~~ sql
+> RESTORE FROM '<backup_location>';
+~~~
 
-In most cases, it's recommended to use the [`BACKUP`][backup] command to take full nightly backups of your cluster:
+{{site.data.alerts.callout_info}}
+Note that a full cluster restore can only be run on a target cluster with no databases or tables.
+{{site.data.alerts.end}}
+
+Or to restore a  database:
 
 {% include copy-clipboard.html %}
 ~~~ sql
-> BACKUP TO '<full_backup_location>';
+> RESTORE DATABASE bank FROM '<backup_location>';
 ~~~
 
-If it's ever necessary, you can then use the [`RESTORE`][restore] command to restore your cluster:
+Or to restore a table:
 
 {% include copy-clipboard.html %}
 ~~~ sql
-> RESTORE FROM '<full_backup_location>';
+> RESTORE TABLE bank.customers FROM '<backup_location>';
 ~~~
 
-### Manual full and incremental backups
+### Full and incremental backups
 
 If your cluster increases to a size where it is no longer feasible to take nightly full backups, you might want to consider taking periodic full backups (e.g., weekly) with nightly incremental backups. Incremental backups are storage efficient and faster than full backups for larger clusters.
 
 Periodically run the [`BACKUP`][backup] command to take a full backup of your database:
 
 {% include copy-clipboard.html %}
 ~~~ sql
-> BACKUP TO '<full_backup_location>';
+> BACKUP TO '<backup_location>';
 ~~~
 
-Then create nightly incremental backups based off of the full backups you've already created.
+Then, create nightly incremental backups based off of the full backups you've already created:
```

Comment:
Can we vet this with someone who doesn't know how "auto append incrementals" works? At face value it might seem like a typo that the command is the same, but on the other hand the note below explains it. I don't know what the alternative is so if it's not confusing to other people then no change needed :)